### PR TITLE
Extract workers

### PR DESCRIPTION
**Pull request by Billy4195**
Extract workers for the routines in arrange_schedule.py main()

### CODE REVIEW

File: arrange_schedule.py [hunk 1]
```diff
@@ -1120,6 +1120,45 @@ def expire_data_check():
...
+    receive_obj = crawler_cwb_img(send_obj)
+    if receive_obj["result"] == "success":
+        "DO NOTHING"
+    else :
+        receive_obj["error"] = "crawler_cwb_img : " + receive_obj["error"]
+        set_system_log(receive_obj)
+
+def do_google_calendar():
+    receive_obj = google_calendar_text()
+    if receive_obj["result"] != "success":
```
Comment (chenyang14) [inline arrange_schedule.py]: The if statement should be `receive_obj["result"] == "success"`

**Comment by Billy4195** [comment]
@chenyang14 fixed, thanks.

File: arrange_schedule.py [hunk 2]
```diff
@@ -1120,6 +1120,14 @@ def expire_data_check():
...
         set_system_log(receive_obj)
 
+def do_google_calendar():
+    receive_obj = google_calendar_text()
+    if receive_obj["result"] == "success":
+        "DO NOTHING"
+    else:
+        receive_obj["error"] = "google_calendar_text" + receive_obj["error"]
+        set_system_log(receive_obj)
+
```
Comment (icecat2012) [inline arrange_schedule.py]: Please add try....catch in google_calendar_text() [line723], or some exception log will not write to set_system_log.

`def google_calendar_text():`
`try:`
[line723~740]
`except Exception as e:`
`return_msg["error"] = str(e)`
`return return_msg`

Comment (Billy4195) [inline arrange_schedule.py]: @icecat2012 fixed.

File: arrange_schedule.py [hunk 3]
```diff
@@ -1315,73 +1350,27 @@ def main():
...
-                else: #Parent
-                    alarm_google_calendar_text = raw_time + 43200.0
-            except:
-                receive_obj["result"] = "fail"
-                receive_obj["error"] = "fork5 error"
-                set_system_log(receive_obj)
-                alarm_google_calendar_text = raw_time + 10.0
+            fork_failed = google_calendar_worker.do(time.strftime('%Y-%m-%dT%H:%M:%SZ',now_time))
+            if fork_failed:
+                alarm_google_calendar_text += 15.0
```
Comment (stvreumi) [inline arrange_schedule.py]: why does this place add 15 rather than 10?

Comment (Billy4195) [reply arrange_schedule.py]: Sorry, I mistakenly change it to 15 seconds.

**Pull request merged**